roles/satellite-clone/vars/satellite_6.9.yml

```diff
@@ -0,0 +1,16 @@
+---
+satellite_package: satellite
+satellite_installer_cmd: satellite-installer
+satellite_scenario: satellite
+capsule_puppet_module: foreman-proxy
+satellite_installer_options: "--foreman-ipa-authentication false --reset-puppet-server-ssl-chain-filepath --disable-system-checks"
```

Comment:
Why are these 2 options always passed in? (I understand disabling system checks, at least party)

Reply:
foreman-ipa-authentication false -> #268 
reset-puppet-server-ssl-chain-filepath -> #349

Reply:
> foreman-ipa-authentication false -> #268

I guess that makes sense.

>  reset-puppet-server-ssl-chain-filepath -> #349

Hah, just last Friday I was playing with this: https://github.com/theforeman/puppet-puppet/pull/779/commits/c05e0a9c8111a7cb27c2f3f25675e5584ff0c261

That said, it very much feels like a 6.3 -> 6.4 change due to Puppet 3 -> 4. In current versions I don't think it would be needed.

Reply:
> Hah, just last Friday I was playing with this: [theforeman/puppet-puppet@c05e0a9](https://github.com/theforeman/puppet-puppet/commit/c05e0a9c8111a7cb27c2f3f25675e5584ff0c261)
> 
Beautiful coincidence!

> That said, it very much feels like a 6.3 -> 6.4 change due to Puppet 3 -> 4. In current versions I don't think it would be needed.

Let's give it a whirl (about to start a test of this PR right now to see)

Reply:
> foreman-ipa-authentication false -> #268
> reset-puppet-server-ssl-chain-filepath -> #349

TYVM. Could you add this as a comment in the YAML so that future us / future somebody else can find it more easily?

Reply:
Can do. I feel a utility such as this (expected to run on old, old versions of Satellite) warrants excessive commenting